### PR TITLE
Remove validation from siret field

### PR DESCRIPTION
**Pull request by fatmaBouchekoua**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.1.x
| Description?  | Some users are blocked with the valdation of the siret field.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? |no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-2754
| How to test?  | Add B2B customer, and try to add a customer from BO.

### CODE REVIEW

File: classes/Customer.php [hunk 1]
```diff
@@ -186,7 +186,7 @@ class CustomerCore extends ObjectModel
             'optin' => array('type' => self::TYPE_BOOL, 'validate' => 'isBool'),
             'website' => array('type' => self::TYPE_STRING, 'validate' => 'isUrl'),
             'company' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
-            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isSiret'),
+            'siret' => array('type' => self::TYPE_STRING, 'validate' => 'isGenericName'),
```
Comment (aleeks) [inline classes/Customer.php]: Hello @fatmaBouchekoua 
Why did you not adapt isSiret function ? https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Validate.php#L1092

WDYT @maximebiloe

Comment (fatmaBouchekoua) [inline classes/Customer.php]: @vincentbz said not to apply any verification on this field.

Comment (aleeks) [inline classes/Customer.php]: Ok, so maybe add a maxlength 14 ? because, it's the max on db_structure..

@vincentbz why do not apply any verification ?!!

**Comment by vincentbz** [comment]
the problem with the verification is that the field is used as siret for france. But if you are in spain, the check is also made on the SIRET, whereas it doesn't exist there.
So it's a nonsense to check something that doesn't exist, and it blocks all the other countries.

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by aleeks** [comment]
I'm closing this, see: #7898 
Regards

**Pull request closed without merging**